onboarding/asset-management-directions.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Directions for information management aspects of the project.
+subtitle: Your efforts will pay off.
+show_sidebar: true
+toc: true
+---
+
+## Reference and literature management
+
+We assume that you have completed the [first steps](/onboarding/#first-steps) and you are part of the NutritionalLungImmunity Zotero group. We ask all our team members to add relevant literature to the Zotero NutritionalLungImmunity group and tag their items with a minimal set of tags. You can read about how to add Zotero tags [here](https://www.zotero.org/support/collections_and_tags). Whatever tags you add will be added to the other team members tag, assuming that you are syncing your library. Please check before adding a paper if the reference is already in the library--if so, simply add tags to the existing reference.
+
+To have the new bibliography be included on the website, you can do one of two things. You can manually export the bib file and drop it and save it as```\_bibliography\references.bib,``` or alternatively, you can use an add-on such as [better BibTex](https://retorque.re/zotero-better-bibtex/) that automatically exports your bib file. Once you are ready to push your new bib file onto the website,  follow [these steps](#github-and-jekyll-workflow) on how to do so. We use [Jekyll Scholar](https://github.com/inukshuk/jekyll-scholar) to manage the reference on the webfront (hence why the standardized tags!). It allows you to include subset of literature on the website based on fields in the bibfile.
+
+### Reference tags
```

Comment:
want to say anything about how to add tags, or if they want to remove or edit tags?